targets/targets.json

```diff
@@ -84,6 +84,10 @@
             "default-adc-vref": {
                 "help": "Default reference voltage for ADC (float)",
                 "value": "NAN"
+            },
+            "flash-non-uniform-sectors": {
```

Comment:
Maybe the opposite is easier to understand:
"flash-uniform-sectors" with default value set to true ? :-)

Reply:
Good suggestion, I agree.

Reply:
👍

Reply:
I'll change it to `internal-flash-uniform-sectors`, because this doesn't apply to external ones like SPI flash.